app/controllers/api/v0/swagger_responses.rb

```diff
@@ -0,0 +1,63 @@
+module Api::V0::SwaggerResponses
+  module AuthenticationErrorId
+    def self.extended(base)
+      base.response 401 do
+        key :description, 'Not authorized.  Happens when API ID or origin are not set.'
```

Comment:
These descriptions are not right for this app (API ID, origin, API token are copy paste and n/a)